amity.py

```diff
@@ -0,0 +1,339 @@
+    occupant1 = Column(String(30), ForeignKey('user.id'))
+    occupant2 = Column(String(30), ForeignKey('user.id'))
+    occupant3 = Column(String(30), ForeignKey('user.id'))
+    occupant4 = Column(String(30), ForeignKey('user.id'))
+    occupant5 = Column(String(30), ForeignKey('user.id'))
+    occupant6 = Column(String(30), ForeignKey('user.id'))
+
+    def __repr__(self):
+        if self.room_type.lower() == 'office':
+            return """<Room(room name='%s', occupant1='%s',
+            occupant2='%s', occupant3='%s', occupant4='%s',
+            occupant5='%s', occupant6='%s')>""" % (self.room_name,
+                                                   self.occupant1,
+                                                   self.occupant2,
+                                                   self.occupant3,
+                                                   self.occupant4,
+                                                   self.occupant5,
+                                                   self.occupant6)
+
+        return """<Room(room name='%s',
+         occupant1='%s',occupant2='%s',
+          occupant3='%s', occupant4='%s')>""" % (self.room_name,
+                                                 self.occupant1,
+                                                 self.occupant2,
+                                                 self.occupant3,
+                                                 self.occupant4)
+
+
+class Amity(object):
+
+    def __init__(self):
+        self.engine = ''
+        self.felo = Fellow()
+        self.staf = Staff()
+        self.livin = LivingSpace()
+        self.offy = Office()
+
+    def connect_db(self, db_name=None):
+        if db_name is not None:
+            name = 'sqlite:///' + db_name[:db_name.index('.')] + '.db'
+        else:
+            name = 'sqlite:///amity.db'
+        return create_engine(name)
+
+    def create_tables(self):
+        engine = self.connect_db()
+
+        Base.metadata.create_all(engine)
+
+    def load_state(self):
+        '''copy data from the database to the application'''
+        engine = self.connect_db()
+
+        print("Please Wait... This may take some few minutes.")
+        # do something
+        Session = sessionmaker(bind=engine)
+        Session.configure(bind=engine)
+        session = Session()
+
+        print("\tReading data from the database")
+        # populate the various lists and dicts
+        self.retrieve_data_from_db(session)
+
+        session.commit()
+        session.flush()
+
+        print("Load State successfully completed")
+
+    def retrieve_data_from_db(self, session):
+        fellows = []
+        staffs = []
+        query_room = session.query(Room).all()
+        query_user = session.query(User).all()
+        for user in query_user:
+            if user.type_person == 'fellow':
+                fellows.append(user.person_name)
+            else:
+                staffs.append(user.person_name)
```

Comment:
`fellows = [user.person_name for user in query_user if user.type_person == 'fellow']`
`staff = [user.person_name for user in query_user if user.type_person == 'staff']`